browser/src/global.d.ts

```diff
@@ -177,6 +177,9 @@ interface Window {
 	createShapesPanel(shapeType: string): HTMLDivElement;
 	initializedUI?: () => void; // initializedUI is an optional function, potentially defined in branding
 	setupToolbar(map: any): void; // TODO should be L.Map
+
+	serviceRoot: string;
+	host: string;
```

Comment:
initialized in https://github.com/CollaboraOnline/online/blob/80b250e30b3dc9b38d1b08e5cf3d03a5e7ff46bc/browser/js/global.js#L377

Reply:
Yes, but this wasn't available for typescript, which this declare then allows.

Reply:
all fine, I just added note for myself as I already found it, just for checking correctness purpose ;)